src/DependencyManagement/DependencyManager.cs

```diff
@@ -0,0 +1,223 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Management.Automation.Language;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.DependencyManagement
+{
+    internal class DependencyManager
+    {
+        // The list of dependent modules for the function app.
+        internal static List<DependencyInfo> Dependencies { get; private set; }
+
+        // This is the location where the dependent modules will be installed.
+        internal static string DependenciesPath { get; private set; }
+
+        // Keep track of whether the function app dependencies have been installed.
+        internal static bool FunctionAppDependenciesInstalled { get; set; }
+
+        // Requirements.psd1 file name.
+        private const string RequirementsPsd1FileName = "requirements.psd1";
+
+        // The list of managed dependencies supported in Azure Functions.
+        internal static readonly List<string> SupportedManagedDependencies = new List<string>() {"Az"};
+
+        // Environment variables to help figure out if we are running in Azure.
+        private const string AzureWebsiteInstanceId = "WEBSITE_INSTANCE_ID";
+        private const string HomeDriveName = "HOME";
+        private const string dataFolderName = "data";

```

Comment:
per convention of naming constant variable it should start with capital case, so please rename this to "DataFolderName"

Reply:
Fixed.